docs/source/reproducers/02-layout.md

```diff
@@ -0,0 +1,97 @@
+# Reproduce CI layout
+The CI Framework allows to reproduce the same layout we're consuming in CI.
+
+While it's not that different from the install_yamls standard layout (one CRC,
+one Compute), the CI layout introduces a third node (ansible controller), and
+all of the nodes have two network interfaces.
+
+## The playbooks
+You will find `reproducer.yml` playbook at the root of the project, as well as
+a specific `scenarios/reproducers` directory.
+
+A second playbook, `reproducer-clean.yml`, allows to remove the created
+resources. Please keep in mind it's destructive, and you won't be able to
+recover anything after the run: virtual machines are destroyed and undefined,
+and the image layers are removed.
+
+### Extra parameter: cifmw_reproducer_repositories
```

Comment:
So we can sync other repos as well, correct?

Reply:
for that specific case, yes. That's the role of that parameter. It's not yet implemented in the "reproduce zuul job", but it will come "soon".